v2/googlecloud-to-neo4j/src/main/java/com/google/cloud/teleport/v2/neo4j/model/helpers/OptionsParamsMapper.java

```diff
@@ -25,12 +25,6 @@ public class OptionsParamsMapper {
   public static OptionsParams fromPipelineOptions(Neo4jFlexTemplateOptions pipelineOptions) {
     OptionsParams optionsParams = new OptionsParams();
     try {
-      if (StringUtils.isNotEmpty(pipelineOptions.getReadQuery())) {
```

Comment:
note: this is technically a breaking change, but these 2 options don't make a lot of sense really...